### PR TITLE
Removed the duplicate extension "js" warning from the NGINX configuration

### PR DESCRIPTION
**Pull request by acataluddi**
The current NGINX configuration triggers the following warning

```
nginx: [warn] duplicate extension "js", content type: "text/javascript", previous content type: "application/javascript" in /etc/nginx/sites-enabled/www.example.com.conf:124
```

### CODE REVIEW

**Comment by acataluddi** [comment]
Hi @susnux,

I've rebased the latest `master` to modernise the branch and highlight the actual code changes.

Kind regards,  
Adriano

**Comment by ChristophWurst** [comment]
the rebase went wrong unfortunately

**Review by susnux — APPROVED** [review]
LGTM, I fixed the rebase (you accidentally merged the master branch in resulted in duplicated commits)

**Comment by susnux** [comment]
@acataluddi but before we can merge this you need to sign you commit.

(pull and `git commit --amend -s`)

**Comment by acataluddi** [comment]
> @acataluddi but before we can merge this you need to sign you commit.

Hi @susnux, done.

Kind regards,
Adriano C.

**Comment by ChristophWurst** [comment]
@nickvergessen you accidentally dropped the sign-off

**Comment by nickvergessen** [comment]
I used the Github button "Update branch".
I don't think it removed anything, but it more was not sign-off-ed before.

**Review by msundin — APPROVED** [review]
Tested ok that this removes the warning.

**Review by susnux — APPROVED** [review]

**Review by miaulalala — APPROVED** [review]

**Comment by miaulalala** [comment]
Hi @acataluddi - seems the signoff was dropped. Can you please checkout your branch, do a pull, rebase on current master and do git commit --amend -s?

**Comment by acataluddi** [comment]
> Hi @acataluddi - seems the signoff was dropped. Can you please checkout your branch, do a pull, rebase on current master and do git commit --amend -s?

Hi Anna,

I've signed off all the commits and pushed the branch for a second review.

Kind regards,
Adriano C.

**Pull request merged**
